hack/generate-cd.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+SOURCE_PATH="$(dirname $0)/.."
+IMAGE_REGISTRY="$(${SOURCE_PATH}/hack/get-image-registry.sh)"
+CD_REGISTRY="$(${SOURCE_PATH}/hack/get-cd-registry.sh)"
+COMPONENT_NAME="$(${SOURCE_PATH}/hack/get-cd-component-name.sh)"
+
+CA_PATH="$(mktemp -d)"
+TMP_COMPONENT_DESCRIPTOR_PATH="${CA_PATH}/component-descriptor.yaml"
+
+if ! which component-cli 1>/dev/null; then
+  echo -n "component-cli is required to generate the component descriptors"
+  echo -n "Trying to installing it..."
+  go get github.com/gardener/component-cli/cmd/component-cli
```

Comment:
I would prefer this to be part of the `install-requirements` target in the Makefile https://github.com/gardener/gardener-extension-os-gardenlinux/blob/0b5c895c3b3596ec3f656c72f156c9983c9c1503/Makefile#L60-L63

Reply:
Installation of component-cli moved to make target install-requirements